torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -888,9 +888,22 @@ Operator(                                                                      \
     DEFINE_INT_FLOAT_OP(aten::remainder, fmod((b + fmod(a, b)), b), float)
 
 
-    // TODO: Support python floordiv (//)
-    // Right now aten::floordiv is only used by loop unrolling
-    DEFINE_INT_OP(aten::floordiv, a / b)
+    // in c++ int division rounds to the integer closer to 0, in python floordiv
+    // rounds to lower integer
+    DEFINE_INT_OP(aten::floordiv, int64_t(std::floor(float(a) / float(b))))
+    DEFINE_INT_FLOAT_OP(aten::floordiv, std::floor(a / b), float)
```

Comment:
You should be able to combine the `int` and `float` versions of this with `DEFINE_GENERIC_OP`